### DISCUSSION TITLE
Used to be able to run ctest from the status bar and no longer can

### DISCUSSION THREAD

**Discussion by av4625** [microsoft/vscode-cmake-tools#3193] [Q&A] [answered]
I used to be able to click run ctest in the status bar and it would build my code and then run all tests added to ctest from with CMake and the output would show in the output at the bottom and would look the same as if I typed `ctest` on the command line (without colour :( ).

<img width="888" alt="image" src="https://github.com/microsoft/vscode-cmake-tools/assets/19472253/99002ee9-b4ba-4ebc-b25d-c68d534ccc0a">

When I click run ctest from the status bar now it opens the testing browser on the left side and I have to choose which tests to run. If I run all the output is separated for each test set. It is not like the output you get if you run `ctest` on the command line, this what the vscode extension used to do. Is there anyway to go back to how it used to work?

Thanks

**Comment by gcampbell-msft** [comment, marked as answer]
@av4625 There are multiple different options here. 

1. There is a "Run Tests" entry point from the Project Status view: 
![image](https://github.com/user-attachments/assets/f9c49e53-e955-406d-8291-4628c9b0af5f)
2. If you have strong feelings about the status view versus the status bar, you can modify the `cmake.options.statusBarVisibility` or the `cmake.options.advanced` settings in order to control where certain buttons show up. 
Thanks!
